docs/1-mechanism/readme.md

```diff
@@ -16,11 +16,11 @@ Even though the Aragon Protocol could theoretically resolve any type of binary d
 
 ### 1.2. High-level flow
 
-- Guardians deposit ANT into a bonding curve to generate ANJ tokens.
-- Guardians stake ANJ to the Protocol contract and schedule their activation and deactivation for the time period in which they can be drafted to rule on disputes.
+- Guardians deposit ANT into a bonding curve to generate ANT tokens.
```

Comment:
I think we can safely remove this for now :)

Reply:
Addressed in 81a4818b240c373ba3faa712d08cc94b7d06a181